lib/topic_view.rb

```diff
@@ -735,6 +721,67 @@ def read_posts_set
 
   private
 
+  def load_mentioned_users
```

Comment:
I just realized something, the only reason we need to load mentioned users is for user status. If the site setting is disable, should we even bother loading mentioned users here?